packages/sdk/src/config/index.ts

```diff
@@ -18,3 +18,5 @@ export const apiConfig = {
     }
   }
 }
+
+export const INSTANT_BUY_SELLER_INPUT_INDEX = 2
```

Comment:
Can we add a comment to explain the rationale behind this magic number?

Reply:
added; https://github.com/sadoprotocol/ordit-sdk/pull/50/commits/c3359ac4a91c44bdb8ba344aa240db782a3b1976